pgrx/src/datum/json.rs

```diff
@@ -67,12 +67,12 @@ impl FromDatum for JsonB {
                 pg_sys::jsonb_out,
                 &[Some(detoasted.into())],
             )
-            .expect("failed to convert jsonb to a cstring");
+            .expect("failed to convert a jsonb to a cstring");
```

Comment:
Likewise, "Datum must refer to JSONB for Postgres to serialize it to a cstring".